utils/proxy.ts

```diff
@@ -2,31 +2,14 @@ import Axios from 'axios'
 import { NFTMetadata } from '@/components/rmrk/service/scheme'
 import { APIKeys, pinFileToIPFS } from '@/utils/pinata'
 import { extractCid, justHash } from '@/utils/ipfs'
-import { saveKey } from '@/utils/cloudflare'
-
-type CdnUpload = {
-  id: string
-  uploadURL: string
-}
-
-type CdnUploadResponse = {
-  result: {
-    id: string
-    filename: string
-    uploaded: string
-    requireSignedURLs: boolean
-    variants: string[]
-  }
-  success: boolean
-}
```

Comment:
This is my only concern 👁️

Reply:
can you elaborate more about this line? it seems like no file used that type. I found similar type in here https://github.com/kodadot/nft-gallery/blob/main/utils/directUpload.ts#L20-L29